src/services/event-processor.service.js

```diff
@@ -48,6 +49,8 @@ class EventProcessorService {
         return
       }
 
+      console.log(`Processing eventId: ${msg.id}`)
```

Comment:
Console.log?

Reply:
There is no logger in this service. console.log statements everywhere. May be we need a story to streamline that.